setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
it is my understanding that this is only used in setuptools/distutils

I would be greatly appreciative of supporting information for this assertion, i would like to minimise unwanted consequences